pkg/filestore/blob_store.go

```diff
@@ -28,9 +28,7 @@ func (s *BlobStore) Init() error {
 
 func (s *BlobStore) WaitForReady(ctx context.Context) error {
 	// there's no waiting, it's either there or it's not. the pod won't come up if the volume didn't mount.
-	volumeMount := "/kotsadmdata"
-	_, err := os.Stat(volumeMount)
-	return err
+	return nil
```

Comment:
what is the reason for this change?

Reply:
When running kotsadm on the host, the paths are different, and this won't work.
Also, this isn't really a readiness check. It's not an error condition that's going to resolve over time, this is here just incase we fail to mount this directory. This interface function feels like it shouldn't really exist since all stores don't need a "wait for". Minio needed it, so we copied it over which makes sense. But short of removing that interface and refactoring, this just didn't make sense as a check that we will "wait for" at startup.